authentik/core/management/commands/repair_permissions.py

```diff
@@ -14,6 +15,11 @@ class Command(BaseCommand):
     @no_translations
     def handle(self, *args, **options):
         """Check permissions for all apps"""
+
+        # Remove potential lingering old permissions
+        # See https://code.djangoproject.com/ticket/28417
+        call_command("remove_stale_contenttypes", "--no-input")
```

Comment:
Instead of using `call_command` this should be directly calling the command from `from django.contrib.contenttypes.management.commands import remove_stale_contenttypes`, also needs to be done in the `with tenant:` context manager